src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1ApplicationUserDetailsEntity.kt

```diff
@@ -16,6 +16,6 @@ class Cas1ApplicationUserDetailsEntity(
   @Id
   val id: UUID,
   val name: String,
-  val email: String,
-  val telephoneNumber: String,
+  val email: String?,
+  val telephoneNumber: String?,
```

Comment:
are these changes temporary, to be reverted after the backfilling migration? If not, should we move these changes back to the earlier commit?

Reply:
This is me trying to avoid a big history rewrite. I'll have a good at fixing that tomorrow. I think the change are permanent because the backfill would never be updated for a submitted application.